### DISCUSSION TITLE
[488310] Eleven Table Tennis [仅本体]

### DISCUSSION THREAD

**Discussion by xiaoning1992** [pjy612/SteamManifestCache#328] [Share]
` appid:488310
` 

```
488311;6a8f5a73fe66474a12587100029fef4b0c2a1344323ae6d0167708fefb00b7b3
```

[488311_3934060744203170595.zip](https://github.com/pjy612/SteamManifestCache/files/13859460/488311_3934060744203170595.zip)

内测码：757CC2366D028DF0
邮箱：[276148226@qq.com](mailto:276148226@qq.com)

**Comment by mgxhl** [comment]
Thanks for sharing, can you update it if it's convenient? I want to test the third person view for video recording purposes. Even if it doesn't update, I might wait for Black Friday to take advantage of the deals to get the app.

**Reply by xiaoning1992** [reply]
Updated

**Reply by Pongsakorn132** [reply]
if got only manifest file how to add game with out lua

**Reply by mgxhl** [reply]
THX!

**Comment by xiaoning1992** [comment]
[488311_503111911515041677.zip](https://github.com/user-attachments/files/17572453/488311_503111911515041677.zip)
2024-10-30 更新Manifest

**Reply by pjy612** [reply]
更新可以直接编辑 原帖子
